Formula/p/pip-audit.rb

```diff
@@ -137,6 +137,10 @@ def install
   end
 
   test do
-    assert_match "No known vulnerabilities found", shell_output("#{bin}/pip-audit --progress-spinner=off 2>&1")
+    (testpath/"requirements.txt").write <<~EOS
+      six==1.16.0
+    EOS
+    output = shell_output("#{bin}/pip-audit --requirement #{testpath}/requirements.txt --progress-spinner=off 2>&1")
```

Comment:
You can also pass in `--no-deps` here to speed things up a bit (it'll only audit `six==1.16.0` in that case, rather than resolving everything under it.)

Reply:
cool, updating it